### PR TITLE
added documentation for VaR application

### PR DESCRIPTION
**Pull request by willb**
This commit adds basic installation and usage documentation for the value-at-risk notebook application.

### CODE REVIEW

**Comment by elmiko** [comment]
this looks nice Will, the lightning talk link will be broken is there going to be a slide deck for this?

**Comment by willb** [comment]
I could make one!  I was thinking that the notebook itself is pretty much self-documenting wrt the approach but could adapt something pretty quickly, I suspect.

**Comment by elmiko** [comment]
either way, if no deck is really needed for this i can make some adjustments to the menu to allow for that case. i'm good with either result.

**Comment by willb** [comment]
Well, I guess I'd argue that the template should be sufficiently parameterized to do the right thing if no deck is present _and_ that a deck for this app is nice-to-have.  ;-)

**Comment by elmiko** [comment]
haha, nice!

File: _applications/var.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+This application characterizes historical returns of stock data and then simulates many possible outcomes for a given portfolio over a given time horizon.  The idea is that you can characterize the value at risk for a given percentage of simulated outcomes (e.g., "there is a 5% chance we'd lose more than $1M USD in the next two weeks").  While you wouldn't want to use the techniques in this application to guide real investment decisions, it also demonstrates how to use a Jupyter notebook with Apache Spark in OpenShift and you can modify the basic simulation to model security returns in a more sophisticated manner.
+
+<h1 id="architecture">Architecture</h1>
+
+The driver for the value-at-risk application is a [Jupyter](http://jupyter.org/) notebook.  This notebook connects to a Spark cluster and schedules jobs to perform simulations.
+
+<h1 id="installation">Installation</h1>
+
+Installing the value-at-risk notebook is straightforward; you simply need to crete an OpenShift project, deploy a Spark cluster in that project, and install and run the notebook image.
```
Comment (elmiko) [inline _applications/var.md]: s/crete/create/

File: _applications/var.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Prerequisites
+
+The easiest way to get a Spark cluster going in your OpenShift project is to use [`oshinko-deploy.sh`](https://github.com/radanalyticsio/oshinko-rest/blob/master/tools/oshinko-deploy.sh).  This provides a streamlined installation for the Oshinko cluster manager into your project.  Oshinko is a service to manage Spark clusters for OpenShift projects.
```
Comment (elmiko) [inline _applications/var.md]: unless we need the rest server, we should probably switch this link to point at the oshinko-webui repository. it has a separate oshinko-deploy.sh which is functionally equivalent to the rest one, but uses a webui with the integrated cli tooling.

**Comment by elmiko** [comment]
thanks for the update Will, lgtm!

**Pull request merged**